docs/src/pages/versions/versions.md

```diff
@@ -17,19 +17,38 @@ You can use it to see what changes are coming and provide better feedback to MUI
 
 MUI follows [Semantic Versioning 2.0.0](https://semver.org/).
 MUI version numbers have three parts: `major.minor.patch`.
 The version number is incremented based on the level of change included in the release.
 
-- **Major releases** contain significant new features, some but minimal developer assistance is expected during the update. When updating to a new major release, you may need to run update scripts, refactor code, run additional tests, and learn new APIs.
+- **Major releases** contain significant new features, some but minimal developer assistance is expected during the update.
+  These releases include [breaking changes](#what-doesnt-count-as-a-breaking-change).
+  When updating to a new major release, you may need to run update scripts, refactor code, run additional tests, and learn new APIs.
 - **Minor releases** contain important new features.
-  Minor releases are fully backward-compatible; no developer assistance is expected during update, but you can optionally modify your apps and libraries to begin using new APIs, features, and capabilities that were added in the release.
+  Minor releases are fully backward-compatible; no developer assistance is expected during the update, but you can optionally modify your apps and libraries to begin using new APIs, features, and capabilities that were added in the release.
 - **Patch releases** are low risk, contain bug fixes and small new features.
-  No developer assistance is expected during update.
+  No developer assistance is expected during the update.
+
+## What doesn't count as a breaking change?
+
+We call "breaking changes" those that require updating your codebase when upgrading to a new version, with the exception of:
+
+- **APIs starting with "unstable\_"**. These are provided as experimental features whose APIs we are not yet confident in.
+  By releasing these with an `unstable_` prefix, we can iterate faster and get to a stable API sooner, or simply learn that we don't need the API/feature in the first place.
+- **APIs documented as experimental**. Same as the above.
+- **Undocumented APIs and internal data structures**. If you access internal properties, there is no warranty. You are on your own.
+- **Development warnings**. Since these don't affect production behavior, we may add new warnings or modify existing warnings in between major versions.
+  In fact, this is what allows us to reliably warn about upcoming breaking changes.
+- **Pre-releases versions**. We provide pre-release versions as a way to test new features early, but we need the flexibility to make changes based on what we learn in the pre-release period.
+  If you use these versions, note that APIs may change before the stable release.
+- **Small CSS changes**. Visual design changes that have a very low probability of negatively impacting your UI are not considered breaking.
+
+In general, we don't release the above changes in patch releases.
```

Comment:
Don't we?

Reply:
It depends on which of the above this sentence applies. It should probably apply all but the last point regarding CSS changes.

Reply:
My idea was that patch releases are ones you can most often blindly upgrade, even if you need nasty things.

Reply:
So whenever we do change to unstable APIs it should be minor version, right?

Reply:
I'm removing this sentence, to use our best judgment for this, but I think that the general idea holds, using new patches should be low risk.